homeassistant/components/mycroft/manifest.json

```diff
@@ -1,10 +1,9 @@
 {
   "domain": "mycroft",
   "name": "Mycroft",
-  "codeowners": [],
-  "disabled": "Dependencies not compatible with the new pip resolver",
+  "codeowners": ["@NeatBasis"],
   "documentation": "https://www.home-assistant.io/integrations/mycroft",
   "iot_class": "local_push",
-  "loggers": ["mycroftapi"],
-  "requirements": ["mycroftapi==2.0"]
+  "loggers": ["mycroft_bus_client"],
+  "requirements": ["mycroft-messagebus-client==0.10.1"]
```

Comment:
This library currently pins a requirement to a specific version. We don't want libraries to do that to avoid problems with package version conflicts.

It's pinning the `pyee` requirement.

https://github.com/MycroftAI/mycroft-messagebus-client/blob/01ade972facb9c2287d4b9eade49b7f8c1e74655/setup.py#L51

https://github.com/MycroftAI/mycroft-messagebus-client/blob/01ade972facb9c2287d4b9eade49b7f8c1e74655/setup.py#L30-L39

https://github.com/MycroftAI/mycroft-messagebus-client/blob/01ade972facb9c2287d4b9eade49b7f8c1e74655/requirements.txt#L2